src/neo4j_genai/components/kg_writer.py

```diff
@@ -18,11 +18,12 @@
 from typing import Literal, Optional
 
 import neo4j
+from pydantic import validate_call
```

Comment:
Any idea about why we still have these imports reordering changes, even after we added the pre-commit hood and the step in the CI?

Reply:
I'm not sure on that, something to investigate into